src/Pate/Equivalence/MemoryDomain.hs

```diff
@@ -49,26 +49,23 @@ import qualified Pate.Parallel as Par
 -- Memory domain
 
 -- | This wrapper around a 'PMC.MemCellPred' describes ranges of memory
--- covered by this domain.  Each entry in 'PMC.MemCellPred'
--- contains the predicate determining whether or not it is "in".
+-- covered by this domain. A memory domain is defined exclusively -
+-- all addresses are considered to be "in" the domain unless they are
+-- present in the given 'PMC.MemCellPred'.
+-- Notably this does not consider the memory region covered by a
+-- given 'PMC.MemCell'. A cell must be exactly included in the
+-- underlying 'PMC.MemCellPred' to be excluded by the domain -
+-- it is not sufficient for a cell to be subsumed by an entry.
 --
--- The interpretation of those predicates is subject to the 'memDomainPolarity'.
+-- This is sound, less precise, and significantly easier to
+-- prove membership.
```

Comment:
What is it less precise than, or significantly easier to prove membership compared to?

Reply:
Good question. I moved this discussion to the function where it is relevant and expanded on it a bit.